crux-mir-comp/src/Mir/Cryptol.hs

```diff
@@ -326,7 +326,11 @@ munge sym shp rv = do
                     W4.justPartExpr sym <$> go shp rv
                 return $ MirVector_PartialVector pv'
             MirVector_Array _ -> error $ "munge: MirVector_Array NYI"
-        -- TODO: StructShape
+        go (StructShape _ _ flds) (AnyValue tr rvs)
+            | StructRepr cr <- tr
```

Comment:
Nit: The naming convention in crux-mir is generally `tpr` for values of type `TypeRepr tp` and `ctx` for `CtxRepr ctx` (a.k.a `Assignment TypeRepr ctx`)